moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -319,11 +319,11 @@ bool ChompOptimizer::optimize()
 {
   bool optimization_result = 0;
   ros::WallTime start_time = ros::WallTime::now();
-  double averageCostVelocity = 0.0;
-  int currentCostIter = 0;
+  // double averageCostVelocity = 0.0;
```

Comment:
Is there a reason to keep these around or can we just delete them?

Reply:
Yes. Down there, is some currently commented code using those variables.